tox.ini

```diff
@@ -20,7 +20,7 @@ commands = nosetests -x -s tests/unittests --cover-html --with-coverage  --cover
 
 [testenv:pep8]
 commands=
-    flake8 network_plugin server_plugin vcloud_plugin_common tests manager_blueprint/scripts
-ignore =
+    flake8 --ignore=E501 network_plugin server_plugin vcloud_plugin_common tests manager_blueprint/scripts
```

Comment:
I trimming line, when it possible. Sometime not suitable trim lines 80 or 82 chars length.